docs/examples/authentication.js

```diff
@@ -50,5 +50,5 @@ const account = { login: '', password: '' };
 
 const verifyIsLoggedIn = async (page) => {
   await page.click('summary[aria-label="View profile and more"]')
-  assert(await page.$(`text="Your profile"`));
+  assert(await page.waitForSelector(`text="Your profile"`));
```

Comment:
It doesn't really make sense to assert here right? waitForSelector is truthy or throws.
```suggestion
 await page.waitForSelector(`text="Your profile"`);
```

Reply:
That's correct, but it is logical to assert in the `verifyIsLoggedIn` function, so why not?